Sources/Defaults/Observation.swift

```diff
@@ -119,95 +119,48 @@ extension Defaults {
 		Thread.current.threadDictionary[key] = false
 	}
 
-	final class UserDefaultsKeyObservation: NSObject, Observation {
-		typealias Callback = (BaseChange) -> Void
-
-		private weak var object: UserDefaults?
-		private let key: String
-		private let callback: Callback
-		private var isObserving = false
+	final class SuiteKeyPair: Hashable {
+		weak var suite: NSObject?
+		let key: String
 
-		init(object: UserDefaults, key: String, callback: @escaping Callback) {
-			self.object = object
+		init(suite: NSObject, key: String) {
```

Comment:
Why not use the specific type?

```suggestion
		init(suite: UserDefaults, key: String) {
```